From 5d1b003772bc9899c291ef25097bfc7860eaaff3 Mon Sep 17 00:00:00 2001
From: Priyanka Bose <priyanka.bose@intel.com>
Date: Wed, 19 Sep 2018 23:16:40 +0530
Subject: [PATCH] Fixing the Capstone Disassembler build to compile for x86_64
 arch variant

Change-Id: Ic7e3c2d1ad44e0a4fa3ffd52ff2b78c0127bec12
Category: device enablement
Domain: AOSP.ART-Other
Origin: internal
Upstream-Candidate: no, proprietary
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-68865
Signed-off-by: Priyanka Bose <priyanka.bose@intel.com>
---
 Android.bp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Android.bp b/Android.bp
index f580d14..c985cc8 100644
--- a/Android.bp
+++ b/Android.bp
@@ -17,6 +17,7 @@ arm64_srcs = [
 ]
 
 x86_flags = ["-DCAPSTONE_HAS_X86"]
+
 x86_srcs = [
     "arch/X86/X86DisassemblerDecoder.c",
     "arch/X86/X86Disassembler.c",
@@ -53,6 +54,11 @@ cc_defaults {
             cflags: arm_flags + arm64_flags + x86_flags,
             srcs: arm_srcs + arm64_srcs + x86_srcs,
         },
+	x86_64: {
+            cflags: arm_flags + arm64_flags + x86_flags,
+            srcs: arm_srcs + arm64_srcs + x86_srcs,
+        },
+
         arm64: {
             cflags: arm_flags + arm64_flags,
             srcs: arm_srcs + arm64_srcs
-- 
1.9.1

